tests/test_cases/test_cocotb/test_handle.py

```diff
@@ -34,8 +34,12 @@ def test_lessthan_raises_error(dut):
 
 
 @cocotb.test()
-def test_bad_attr(dut):
-    yield cocotb.triggers.NullTrigger()
+async def test_bad_attr(dut):
+    await cocotb.triggers.NullTrigger()
```

Comment:
```suggestion
```

This only existed previously to make it a valid generator. It isn't doing anything here.